Composer/packages/client/src/recoilModel/atoms/botState.ts

```diff
@@ -370,6 +371,11 @@ export const isEjectRuntimeExistState = atomFamily<boolean, string>({
   default: false,
 });
 
+export const currentTargetState = atomFamily<PublishTarget, string>({
```

Comment:
currentPublishTargetState is a more descriptive name

Reply:
replaced